.github/workflows/release.yml

```diff
@@ -290,6 +272,63 @@ jobs:
           asset_name: k0s-airgap-bundle-${{ needs.release.outputs.tag_name }}-arm
           asset_content_type: application/octet-stream
 
+
+  build-image:
+    needs:
+      - release
+      - x64
+      - arm64
+      - armv7
+    runs-on: [self-hosted,linux,x64]
+    steps:
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Login to DockerHub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+
+      - name: Fetch k0s amd64
+        uses: actions/download-artifact@v2
+        with:
+          name: k0s-amd64
```

Comment:
I don't get how `name: k0s-amd64` maps into the correct binary as the artifacts have name like `k0s-v1.23.1-rc.2+k0s.1-amd64`?

Reply:
It's just build artifacts (not those that will be posted to the release), so they have [simple names](https://github.com/k0sproject/k0s/pull/1442/files#diff-87db21a973eed4fef5f32b267aa60fcee5cbdf03c67fafdc2a9b553bb0b15f34R199) like `k0s-arm64`